tests/plasma/test_grids.py

```diff
@@ -505,6 +505,12 @@ def uniform_cartesian_grid():
     return grid
 
 
+def test_soften_edges(uniform_cartesian_grid):
+    grid = uniform_cartesian_grid
+    grid.soften_edges()
+    assert grid["x"][0, 0, 0] == 0
```

Comment:
```suggestion
    u.assert_quantity_allclose(grid["rho"][0, :, :], 0)
    u.assert_quantity_allclose(grid["rho"][:, 0, :], 0)
    u.assert_quantity_allclose(grid["rho"][:, :, 0], 0)
    u.assert_quantity_allclose(grid["rho"][-1, :, :], 0)
    u.assert_quantity_allclose(grid["rho"][:, -1, :], 0)
    u.assert_quantity_allclose(grid["rho"][:, :, -1], 0)
```

🧪 I'm wondering if we could use [`astropy.tests.helper.assert_quantity_allclose`](https://docs.astropy.org/en/latest/api/astropy.tests.helper.assert_quantity_allclose.html) to help check that all of the grid faces are zeroed out. 

I changed it to `"rho"` instead of `"x"` so that it's checking a physical quantity rather than a coordinate.  I haven't tried this, so the latter arguments might need to be changed to `0 * u.kg * u.m**-3` if `grid["rho"]` is a `Quantity`.

We could potentially parameterize this, but I don't know how to do that with the `:` indices. 🤔